hilgendorf_moveit_config/config/joint_limits.yaml

```diff
@@ -0,0 +1,61 @@
+joint_limits:
+  right_wrist_2_joint:
+    has_velocity_limits: true
+    max_velocity: 3.2
+    has_acceleration_limits: false
+    max_acceleration: 0
```

Comment:
We have had quite some issues with the UR controller throwing *safety stop* errors with whatever `max_acceleration` values we configured. The 'old' behaviour of the MSA was to set `max_acceleration` to 1/5th of `max_velocity`. I'm not sure what the best value here would be. Perhaps wait for some more opinions.